.github/workflows/build-docker-image.yml

```diff
@@ -2,45 +2,41 @@ name: Build multi-arch Docker Image
 
 on:
   release:
-    types: [published]
+    types: [ published ]
   push:
-    branches: [master]
+    branches: [ master ]
   pull_request:
 
 jobs:
-  build:
+  build-linux-and-windows-multiarch:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Login to Docker Hub
-      # Only if we need to push an image
-      if: ${{ github.event_name == 'release' && github.event.action == 'published' }}
-      uses: docker/login-action@v1
-      with:
-        username: ${{ secrets.DOCKERHUB_USERNAME }}
-        password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - uses: actions/checkout@v2
+      - name: Login to Docker Hub
+        # Only if we need to push an image
+        if: ${{ github.event_name == 'release' && github.event.action == 'published' }}
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
 
-    # Setup for buildx
-    - name: Set up QEMU
-      uses: docker/setup-qemu-action@v1
-    - name: Set up Docker Buildx
-      id: buildx
-      uses: docker/setup-buildx-action@v1
+      # Setup for buildx
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
 
-    # Debugging information
-    - name: Docker info
-      run: docker info
-    - name: Buildx inspect
-      run: docker buildx inspect
+      # Debugging information
+      - name: Docker info
+        run: docker info
+      - name: Buildx inspect
+        run: docker buildx inspect
 
-    # Build and (optionally) push the image
-    - name: Build image
-      uses: docker/build-push-action@v2
-      with:
-        context: .
-        file: ./Dockerfile
-        platforms: linux/amd64,linux/arm/v7,linux/arm64,linux/ppc64le,linux/s390x
-        # Only push if we are publishing a release
-        push: ${{ github.event_name == 'release' && github.event.action == 'published' }}
+      # In case linux/s390x fails with tls issues:
+      # see https://github.com/testcontainers/moby-ryuk/pull/26
+      # and https://github.com/testcontainers/moby-ryuk/pull/40
+      - name: Cross Build for Linux and Windows
         # Use a 'temp' tag, that won't be pushed, for non-release builds
-        tags: testcontainers/ryuk:${{ github.event.release.tag_name || 'temp' }}
+        run: |
+          ./build-multiarch.sh testcontainers/ryuk:${{ github.event.release.tag_name || 'temp' }} ${{ (github.event_name == 'release' && github.event.action == 'published' && 'yes') || 'no' }}
+...
```

Comment:
Is this line a left-over or put there on purpose?

Reply:
I'm not going to be picky about that: we are not implementing an automated, strict YAML validator, which it could be the case at some point. Then, I'd remove them for simplicity of the YAML file